libs/logic-apps-shared/src/utils/src/lib/models/template.ts

```diff
@@ -57,3 +57,20 @@ export interface TemplateContext {
   workflowAppName?: string;
   isMultiWorkflow: boolean;
 }
+
+interface ContentInfo<T> {
```

Comment:
can we export this?

Reply:
Can you elaborate on this @preetriti1 ? This is a public contract should be able to be referenced right?

Reply:
I can explain if you do that in datafetcher :)